_posts/2023-01-12-January-Update-Finishing-my-Igalia-CE.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "January Update: Finishing my Igalia CE"
+date: 2023-01-12
+author: Maíra Canal
+layout: post
+permalink: /january-update-finishing-my-igalia-ce/
+categories: Genel
+tags: [igalia, graphics]
+---
+
+2022 really passed by fast and I'm again finishing another mentorship project.
+I'm finishing my [Igalia Coding
+Experience](https://www.igalia.com/coding-experience/) project this month and I
+had the best experience during those four months. I learned tremendously about
+the Linux graphics stack and now I can say for sure that I would love to keep
+working in the DRM community.
+
+While GSoC was, for me, an experience to get a better understanding of what open
+source is, Igalia CE was an opportunity for me to mature my knowledge of
+technical concepts.
+
+So, this is a summary report of my journey at the Igalia CE.
+
+# IGT tests to V3D
+---
+
+Currently, V3D only has three basic IGT tests: `v3d_get_bo_offset`,
+`v3d_get_param`, and `v3d_mmap`. So, the basic goal of my CE project was to add
+more tests to the V3D driver.
+
+During the CE, I was able to develop tests for almost all eleven V3D ioctls
+(except `v3d_submit_tfu`). I began writing tests to the `v3d_create_bo` ioctl
+and *Performance Monitor* (perfmon) related ioctls. I developed tests that test
+the basic functionality of the ioctls and I inspected the kernel code to
+understand situations where the ioctl should fail.
+
+After those tests, I got the biggest challenge that I had on my CE project:
+performing a Mesa's noop job on IGT. The main problem I faced on submitting a
```

Comment:
In addition to André's suggestions, I'd already say here the reason you need this kind of job. Why a noop job and what you would do with it.